app/views/nominate_practices/index.html.erb

```diff
@@ -1,23 +1,23 @@
 <section class=" grid-container">
   <div class="grid-row grid-col-12 tablet:grid-col-10">
     <div class="grid-col-12 display-inline-block">
-      <h1 class="font-sans-2xl margin-bottom-4">Nominate a practice</h1>
+      <h1 class="font-sans-2xl margin-bottom-4">Nominate an innovation</h1>
       <div class="font-sans-lg line-height-36">
         <span>
-          If you are interested in submitting a practice to the Diffusion Marketplace, review the criteria below and apply through the link.
+          If you are interested in submitting an innovation to the Diffusion Marketplace, review the criteria below and apply through the link.
         </span>
       </div>
       <h2 class="font-sans-xl margin-bottom-405">Criteria</h2>
       <div class="font-sans-sm">
-        <p class="margin-bottom-1">To qualify, your practice must be:</p>
+        <p class="margin-bottom-1">To qualify, your innovation must be:</p>
         <ul class="padding-left-2 margin-top-0 line-height-sans-505">
           <li>Adopted at two or more locations</li>
           <li>Endorsed by a senior executive stakeholder</li>
           <li>An active practice</li>
         </ul>
       </div>
       <a target="_blank"
-         href="mailto:marketplace@va.gov?subject=<%= URI.encode('Nominate a Practice')%>&body=<%= URI.encode("I am writing to nominate a practice for the Diffusion Marketplace.\n\nName of practice:\nOriginating facility:\nPoint of contact:\nSenior executive stakeholder (if known):\nAdoptions (if known):")%>"
+         href="mailto:marketplace@va.gov?subject=<%= URI.encode('Nominate an Innovation')%>&body=<%= URI.encode("I am writing to nominate a practice for the Diffusion Marketplace.\n\nName of practice:\nOriginating facility:\nPoint of contact:\nSenior executive stakeholder (if known):\nAdoptions (if known):")%>"
```

Comment:
practice usage here

Reply:
Got it - both instances.
![image](https://user-images.githubusercontent.com/60527788/136101745-8a9c9829-0b07-4e9b-8493-d063700bdb44.png)